runtime/debug/tracing-adapter.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright (c) 2018 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import devtoolsChannelProvider from './devtools-channel-provider.js';
+import Tracing from '../../tracelib/trace.js';
+
+let streamingToDevtools = false;
+
+function enableTracingAdapter() {
+  if (!streamingToDevtools) {
+    if (!Tracing.enabled) Tracing.enable();
+
+    const channel = devtoolsChannelProvider.get();
+
+    channel.send({
+      messageType: 'trace-time-sync',
+      messageBody: {
+        traceTime: Tracing.now(),
```

Comment:
What is this for? Perhaps we should use high performance timing everywhere.

Reply:
As discussed offline, this is used to map timestamps used by tracing (which is page-load based in the browser and arbitrarily based on node) into human clock.